autoware_launch/config/system/pipeline_latency_monitor/pipeline_latency_monitor.param.yaml

```
@@ -0,0 +1,57 @@
+      map_based_prediction:
+        topic: # input topic with the timestamp and the latency duration of the processing step
+          /perception/object_recognition/prediction/map_based_prediction/debug/processing_time_ms
+        topic_type: # type of the input topic
+          autoware_internal_debug_msgs/msg/Float64Stamped
+        timestamp_meaning: "start" # whether the timestamp represent the "start" or "end" time of the step
+        latency_multiplier: 1.0  # multiplier to convert the input latency duration into ms (e.g., 1e3 for s->ms, 1e-3 for us->ms)
+      multi_object_tracker:
+        topic: # input topic with the timestamp and the latency duration of the processing step
+          /perception/object_recognition/tracking/multi_object_tracker/debug/meas_to_tracked_object_ms
+        topic_type: # type of the input topic
+          autoware_internal_debug_msgs/msg/Float64Stamped
+        timestamp_meaning: "start" # whether the timestamp represent the "start" or "end" time of the step
+        latency_multiplier: 1.0 # multiplier to convert the input latency duration into ms (e.g., 1e3 for s->ms, 1e-3 for us->ms)
+      decorative_object_merger_node:
+        topic: # input topic with the timestamp and the latency duration of the processing step
+          /perception/object_recognition/tracking/decorative_object_merger_node/debug/processing_time_ms
+        topic_type: # type of the input topic
+          autoware_internal_debug_msgs/msg/Float64Stamped
+        timestamp_meaning: "start" # whether the timestamp represent the "start" or "end" time of the step
+        latency_multiplier: 1.0 # multiplier to convert the input latency duration into ms (e.g., 1e3 for s->ms, 1e-3 for us->ms)
+      planning:
+        topic: # input topic with the timestamp and the latency duration of the processing step
+          /planning/planning_validator/validation_status
+        topic_type: # type of the input topic
+          autoware_planning_validator/msg/PlanningValidatorStatus
+        timestamp_meaning: "start" # whether the timestamp represent the "start" or "end" time of the step
+        latency_multiplier: 1.0e3 # multiplier to convert the input latency duration into ms (e.g., 1e3 for s->ms, 1e-3 for us->ms)
+      control:
+        topic: # input topic with the timestamp and the latency duration of the processing step
+          /control/control_system_latency
+        topic_type: # type of the input topic
+          autoware_internal_debug_msgs/msg/Float64Stamped
+        timestamp_meaning: "start" # whether the timestamp represent the "start" or "end" time of the step
+        latency_multiplier: 1.0e3 # multiplier to convert the input latency duration into ms (e.g., 1e3 for s->ms, 1e-3 for us->ms)
+
+    latency_offsets_ms:  # [ms] extra offset to add to the total latency
+      - 0.0 # sensing
+      - 1.4 # perception
+      - 50.0 # planning
+      - 0.0 # control
```

The controller_node operates on a 30-millisecond timer-driven basis. So, 15 ms of offset should be added
```suggestion
      - 15.0 # control
```